buildpack/rewriteLayer.go

```diff
@@ -88,7 +88,9 @@ func rewriteLayer(layer v1.Layer, oldID, newID, oldVersion, newVersion string) (
 		}
 	}
 
-	return tarball.LayerFromReader(b)
+	return tarball.LayerFromOpener(func() (io.ReadCloser, error) {
```

Comment:
was the API deprecated? is it 100% equivalent?

Reply:
Yes, it was deprecated. That's my understanding, that it's the same.

https://pkg.go.dev/github.com/google/go-containerregistry/pkg/v1/tarball#LayerFromReader

Reply:
Although, I didn't look at the full method to see if things could be rewritten around this API change. I was just looking for the smallest change set.